_compdemos/motuz.md

```diff
@@ -42,11 +42,29 @@ this bucket. See [the documentation](/scicomputing/access_credentials/#amazon-we
 
 In Motuz, click the `Cloud Connections` link at the top of the browser window. Then click the `New Connections` button. 
 
-![](/assets/motuz/2019-08-23-12-30-33.png)
 
-In the `Basic` section, set `Type` to `Amazon Simple Storage Service (s3)`. Give the connection a name such as `lab s3 bucket`. Optionally enter the bucket name (`fh-pi-doe-j-eco` here is just an example).
+There are two kinds of credentials - Single Sign-On (SSO) and long-term. By default (since 2024), all users can use SSO credentials, which last for 12 hours. If you need longer-lasting credentials, such as for a long-running transfer, please contact the Cloud Team by emailing `helpdesk`. 
```

Comment:
I'm concerned about terminology- will CLD understand a request for "long term credentials"?  Would it be better to align this with the AWS term (e.g. "IAM Access Key Pair")?